android-artifacts.gradle

```diff
@@ -0,0 +1,22 @@
+uploadArchives {
+    /* task androidJavadocs(type: Javadoc) {
+       source = android.sourceSets.main.java.sourceFiles
+   }
+
+   task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+       classifier = 'javadoc'
+       //basename = artifact_id
+       from androidJavadocs.destinationDir
+   }*/
+
+    task androidSourcesJar(type: Jar) {
+        classifier = 'sources'
+        //basename = artifact_id
+        from android.sourceSets.main.java.sourceFiles
+    }
+
+    artifacts {
+        archives androidSourcesJar
+        //archives androidJavadocsJar
```

Comment:
I left it in there in case someday I will find what it was and this file
was a simple copy and paste from the master file that I have.

On Fri, Jan 23, 2015 at 10:35 AM, Andrew Grosner <
andrew.grosner@raizlabs.com> wrote:

> Dylan, I am not sure what the correct gradle variable was and never got
> around to finding it out. But by running that command, the gradle build
> will fail.
>
> On Fri, Jan 23, 2015 at 10:33 AM, dylanrjames <notifications@github.com>
> wrote:
>
>> In android-artifacts.gradle
>> <https://github.com/Raizlabs/RZAndroidBaseUtils/pull/6#discussion_r23456510>
>> :
>>
>> > +
>> > +   task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
>> > +       classifier = 'javadoc'
>> > +       //basename = artifact_id
>> > +       from androidJavadocs.destinationDir
>> > +   }*/
>> > +
>> > +    task androidSourcesJar(type: Jar) {
>> > +        classifier = 'sources'
>> > +        //basename = artifact_id
>> > +        from android.sourceSets.main.java.sourceFiles
>> > +    }
>> > +
>> > +    artifacts {
>> > +        archives androidSourcesJar
>> > +        //archives androidJavadocsJar
>>
>> Any reason to leave this commented stuff here? And any reason why we
>> wouldn't build Javadocs? Don't we need to do this for Javadoc to be visible
>> from consuming code?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Raizlabs/RZAndroidBaseUtils/pull/6/files#r23456510>.
>>
>
>
>
> --
> --
>
> *Andrew Grosner, Android Developer*
>
> Mobile Strategy, Design & Development
>
> www.raizlabs.com | Check my calendar
> <https://www.google.com/calendar/embed?src=andrew.grosner@raizlabs.com&ctz=America/New_York&mode=WEEK>
>  | @agrosner <http://www.twitter.com/agrosner> | 716-472-5075
> [image: Raizlabs]
>



-- 
--

*Andrew Grosner, Android Developer*

Mobile Strategy, Design & Development

www.raizlabs.com | Check my calendar
<https://www.google.com/calendar/embed?src=andrew.grosner@raizlabs.com&ctz=America/New_York&mode=WEEK>
 | @agrosner <http://www.twitter.com/agrosner> | 716-472-5075
[image: Raizlabs]